docs/streams/quickstart.md

```diff
@@ -123,16 +123,24 @@ const client = new jsc8("https://gdn.paas.macrometa.io");
 <TabItem value="py" label="Python">
 
 ```py
+from operator import concat
+import base64
+import json
+import warnings
 from c8 import C8Client
+import six
+warnings.filterwarnings("ignore")
+
+URL = "gdn.paas.macrometa.io"
+GEO_FABRIC = "_system"
+API_KEY = "my API key" #Change this to my API key
```

Comment:
we should specify the apikey sample as API_KEY = "apikey <yourapikey>" since "apikey" keyword is needed here and the user doesn't know that.

Reply:
We should probably test this together.
For me, it worked fine without the "apikey" keyword.

Reply:
yup it works fine actually